sdk/storage/azure-storage-blob-batch/README.md

```diff
@@ -70,6 +70,10 @@ BlobBatchClient blobBatchClient = new BlobBatchClientBuilder(blobServiceClient).
 
 <!-- embedme ./src/samples/java/com/azure/storage/blob/batch/ReadmeSamples.java#L46-L48 -->
 ```java
+List blobUrls = new ArrayList<>();
+blobUrls.add(blobClient1.getBlobUrl());
+blobUrls.add(blobClient2.getBlobUrl());
+blobUrls.add(blobClient3.getBlobUrl());
```

Comment:
Reason the PR build fails is that these README code samples are generated based on the line range listed in the `embedme` comment.

You'll want to add the new lines of code into that section and update the line range references. To run the validation locally you'll want to run the following:

`mvn package -DskipTests -Dverify-readme{-windows} -f {target POM}`

Use `-Dverify-readme-windows` if you're using a Windows machine, otherwise use `-Dverify-readme`.